### ISSUE TITLE
[Bug] App reports I'm offline.

### ISSUE THREAD

**Issue by nsarzyns** [YTVanced/Vanced#1490]
**Bug description:**
Whenever I launch the vanced app, I'm displayed a message saying to connect to the internet and that I'm offline. The regular YouTube app is fine. I uninstalled and reinstalled the app with the same issue occurring.

**Have you read vanced guides already? (YES/No)**
Yes

**Variant:**
Non-Root

**Vanced version:**
16.29.39

**Link used to download vanced:**
https://vancedapp.com/

**Android version:**
Android 11 

**Device:**
Samsung Galaxy S10+

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
2. Message is displayed that I'm offline

**Expected behaviour:**
App launches to my suggested videos instead of reporting I'm offline.

https://ibb.co/9ctQJfn

**Comment by milindgoel15** [comment]
You wrote that u read vanced guides and yet here u are, reporting the issue. Damn people 

#786

**Comment by nsarzyns** [comment]
I appreciate the assumption! I did read it and followed the steps outlined for the exact issues with no luck. Granted I didn't directly say I followed such steps. But regardless that didn't fix the issue, should've said that as well.

Ended up being a VPN related issue. Only discovered once some other apps lacked access as well. Vanced is allowed around the VPN as well as other things, between yesterday and today something changed and such settings needed to be reapplied.

Not an issue with Vanced but also not the way to respond to an issue tracker
